solitude/base.py

```diff
@@ -151,6 +152,13 @@ def mozilla_error(self, content, field):
     def braintree_error(self, content, field):
         return [f['code'] for f in content['braintree'][field]]
 
+    def print_as_json(self, data):
+        """
+        A utility function used to dump the json to stdout. Useful in tests
+        when you want some output for documentation.
+        """
+        print json.dumps(data, indent=2, cls=JSONEncoder)
```

Comment:
I found it really handy to copy/paste some json from the docs into a test but I had to reformat it to four space indents because of flake8. I don't know if it will cause docs to scroll off the page but maybe set it to four spaces for that reason?